app/main.go

```diff
@@ -42,22 +42,7 @@ var opts struct {
 		Jitter   bool          `long:"jitter" env:"JITTER" description:"jitter"`
 	} `group:"repeater" namespace:"repeater" env-namespace:"CRONN_REPEATER"`
 
-	Notify struct {
-		EnabledError       bool          `long:"enabled-error" env:"ENABLED_ERROR" description:"enable email notifications on errors"`
-		EnabledCompletion  bool          `long:"enabled-complete" env:"ENABLED_COMPLETE" description:"enable completion notifications"`
-		ErrorTemplate      string        `long:"err-template" env:"ERR_TEMPLATE" description:"error template file"`
-		CompletionTemplate string        `long:"complete-template" env:"COMPLET_TEMPLATE" description:"completion template file"`
-		SMTPHost           string        `long:"smtp-host" env:"SMTP_HOST" description:"SMTP host"`
-		SMTPPort           int           `long:"smtp-port" env:"SMTP_PORT" description:"SMTP port"`
-		SMTPUsername       string        `long:"smtp-username" env:"SMTP_USERNAME" description:"SMTP user name"`
-		SMTPPassword       string        `long:"smtp-password" env:"SMTP_PASSWORD" description:"SMTP password"`
-		SMTPTLS            bool          `long:"smtp-tls" env:"SMTP_TLS" description:"enable SMTP TLS"`
-		SMTPTimeOut        time.Duration `long:"smtp-timeout" env:"SMTP_TIMEOUT" default:"10s" description:"SMTP TCP connection timeout"`
-		From               string        `long:"from" env:"FROM" description:"SMTP from email"`
-		To                 []string      `long:"to" env:"TO" description:"SMTP to email(s)" env-delim:","`
-		MaxLogLines        int           `long:"max-log" env:"MAX_LOG" default:"100" description:"max number of log lines name"`
-		HostName           string        `long:"host" env:"HOSTNAME" description:"host name running cronn"`
-	} `group:"notify" namespace:"notify" env-namespace:"CRONN_NOTIFY"`
+	Notify notify.Params `group:"notify" namespace:"notify" env-namespace:"CRONN_NOTIFY"`
```

Comment:
I don't like the way it uses externally defined struct for CLI params. It has two problems - reading/following this is hard and we need to go to other place to figure what are params here, and it also causes an extra coupling, i.e. if we add a cli param what should be processed by main it will go directly to notifier params as well.

To me, mapping params from cli to dedicated notify params feels more apropriate. 
```suggestion
	Notify notify.Params `group:"notify" namespace:"notify" env-namespace:"CRONN_NOTIFY"`
```
```suggestion
	Notify notify.Params `group:"notify" namespace:"notify" env-namespace:"CRONN_NOTIFY"`
```